x-pack/plugins/observability/public/hooks/slo/use_create_slo_rq.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
can you please create a new folder in apps with sub folders as sli, slo etc, i think that will simplify code reviews. 

And perhaps create sub directories as components/hooks etc. The root hooks folder etc structure was meant for shared stuff across plugins, apm, uptime, logs etc. 

Also out of curiosity, we have a useFetcher hook in the same folder. wasn't that enough for fetching data?
what are the special feature of react-query that are being utilised?

Reply:
Hey Shazad, everything (components, hooks) are already in separate 'SLO' named folders.

React Query is not a data fetching library so it is not an equivalent to `useFetcher`.

In terms of what React Query can bring:
- less boilerplate (see the diff)
- caching / invalidating per request,
- caching / invalidating requests in different parts of the app so less prop plumbing
- better type safety
- standardization (the data fetching hooks in `o11y` differ quite a bit in terms of return signature)
- built in features such as refetch on window focus, polling, optimistic updates (some of which I use in this PR).

Reply:
i wonder why it's triggering the observability-ui reviews then? i think instead of creating sli/slo directory inside other sub directories. You should create an app directory inside apps/actionable-observability/ and that should contain all the relevant components. hooks etc sub directories.

Reply:
Because I've made one change in `x-pack/plugins/observability/public/application/index.tsx` which is owned by `obs-ui`.

Reply:
it says i own 14 files in the PR, not just index file :)

Reply:
<img width="1758" alt="image" src="https://user-images.githubusercontent.com/3505601/220301128-ae7e7920-f4d9-4ccb-9bf8-641493ab574b.png">
 

hooks/slo codeowner needs to be updated. this is why i am requesting to create a new app directory here 
like 
`https://github.com/elastic/kibana/tree/main/x-pack/plugins/observability/public/apps/actionable_observability`

and keep everything related inside. Updating each new dir code-owner doesn't seems ideal.

Reply:
Or we could just update the CODEOWNERS file right now and be done with it. I'll do that now.

Reply:
Updated. Let me know if I missed anything that might still trigger your notifications.

Reply:
I assume your concerns have been addressed by https://github.com/elastic/kibana/pull/151672. Resolving this.